### ISSUE TITLE
Can't click the OS X dock icon to open a new window

### ISSUE THREAD

**Issue by havthgem** [atom/atom#9567]
Program startup.
Close the window and then click the dock icon to open, can not generate a new window, just empty(no one).

**Comment by nathansobo** [comment]
@havthgem Can you see any errors in the development tools or anything to go on?

**Comment by samrayner** [comment]
I can't see any errors in developer tools (as there is no window for developer tools to run in) but this is very easy for me to reproduce on OS X 10.11.1 - open atom 1.2.0 > close every window by clicking close top left > click atom icon in dock > nothing

**Comment by nathansobo** [comment]
Thanks. I re-titled the issue now that I understand the report better.

**Comment by livelazily** [comment]
In the system console, atom print this error message while open faild :
```
15/11/14 21:49:02.529 Atom[99613]: Cannot read property 'preventDefault' of undefined
15/11/14 21:49:02.529 Atom[99613]: TypeError: Cannot read property 'preventDefault' of undefined
    at EventEmitter.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/browser/atom-application.js:412:16)
    at emitNone (events.js:67:13)
    at EventEmitter.emit (events.js:166:7)
    at EventEmitter.<anonymous> (/Applications/Atom.app/Contents/Resources/atom.asar/browser/api/lib/app.js:95:19)
    at emitTwo (events.js:87:13)
    at EventEmitter.emit (events.js:172:7)
```

**Comment by Fed03** [comment]
:+1:

**Comment by mnquintana** [comment]
Looks like this is still an issue – confirmed on Atom 1.4.0

**Comment by joshaber** [comment]
This is fixed in 1.5.0-beta0 :metal:
